server.ts

```diff
@@ -68,10 +69,10 @@ app.prepare().then(() => {
                 roleSid: permission,
             });
         } catch (error) {
-            if (error.code === 20404) {
+            if (error.code === userDoesntExist) {
```

Comment:
rename to `ERROR_CODE__NOT_FOUND`, add link to `https://www.twilio.com/docs/api/errors/20404` in comment